src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/Guard.java

```diff
@@ -88,7 +90,7 @@
      * @param forImport the package to check.
      * @return a result {@link AccessResult} indicating whether it can be used.
      */
-    AccessResult verifyImport(final String forImport) {
+    @VisibleForTesting AccessResult verifyImport(final String forImport) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method verifyImport() has an NPath complexity of 18](https://www.codacy.com/app/checkstyle/checkstyle/file/1099998019/issues/source?bid=2193633&fileBranchId=2892283#l93)